src/website/views/components/forms/radio-button/_examples.pug

```diff
@@ -236,22 +273,40 @@ p.-text
             .chi-popover__content
               p.chi-popover__text
                 | Helpful information goes here.
-      .chi-form__item.-mb--1
-        .chi-radio
-          input(class="chi-radio__input", type="radio", name="radios", id="radio-he1")
-          label.chi-radio__label(for="radio-he1") Option 1
-      .chi-form__item
-        .chi-radio
-          input(class="chi-radio__input", type="radio", name="radios", id="radio-he2")
-          label.chi-radio__label(for="radio-he2") Option 2
+      div.chi-form__item
+        chi-radio-button(id="example_help_opt_1" label="Option 1" name="radios")
+      div.chi-form__item
+        chi-radio-button(id="example_help_opt_2" label="Option 2" name="radios")
   .example-tabs.-pl--2
     ul.chi-tabs#example-help
-      li.-disabled
-        a(href=`#webcomponent-help`) Web Component
       li.-active
+        a(href=`#webcomponent-help`) Web Component
+      li
         a(href=`#html-help`) HTML Blueprint
-  .chi-tabs-panel#webcomponent-help
-  .chi-tabs-panel.-active#html-help
+  .chi-tabs-panel.-active#webcomponent-help
+    :code(lang="html")
+      <fieldset>
+        <div class="chi-label__wrapper">
+          <legend class="chi-label">Select an option</legend>
+          <div class="chi-label__help">
+            <button class="chi-button -icon -xs -flat" id="example__help-button" aria-label="Help" data-target="#example__help-popover">
+              <i class="chi-icon icon-circle-info-outline" aria-hidden="true"></i>
+            </button>
+            <section class="chi-popover chi-popover--top -animated" id="example__help-popover" aria-modal="true" role="dialog" aria-hidden="true" x-placement="top">
+              <div class="chi-popover__content">
+                <p class="chi-popover__text">Helpful information goes here.</p>
+              </div>
+            </section>
+          </div>
+        </div>
+        <div class="chi-form__item">
+          <chi-radio-button id="example_help_opt_1" label="Option 1" name="radios"></chi-radio-button>
+        </div>
+        <div class="chi-form__item">
+          <chi-radio-button id="example_help_opt_2" label="Option 2" name="radios"></chi-radio-button>
+        </div>
+      </fieldset>
```

Comment:
The help icon is not working here. Can we use chi-label `info-icon` / `info-icon-message`?

```js
<chi-label ... info-icon info-icon-message="Helpful information goes here.">Label</chi-label>
```

Reply:
I've changed but it works for me